public/js/definitions-html5-css.js

```diff
@@ -0,0 +1,91 @@
+$(document).ready(function(){
```

Comment:
This file should be in client/commonFramework folder and follow the init pattern established there